docs/src/main/asciidoc/building-native-image.adoc

```diff
@@ -90,7 +90,7 @@ Installing via scoop will do this for you.
 ${GRAALVM_HOME}/bin/gu install native-image
 ----
 +
-Some previous releases of GraalVM included the `native-image` tool by default.  This is no longer the case; it must be installed as a second step after GraalVM itself is installed. Note: there is an outstanding issue xref:graal-and-catalina[using GraalVM with macOS Catalina].
+Some previous releases of GraalVM included the `native-image` tool by default.  This is no longer the case; it must be installed as a second step after GraalVM itself is installed. Note: there is an outstanding issue xref:#graal-and-catalina[using GraalVM with macOS Catalina].
```

Comment:
Same here.

Reply:
This is probably only my personal convention, the # is optional. I'll update the PR.